src/main/java/com/company/Main.java

```diff
@@ -41,18 +59,16 @@ private static Matrix enhancedThreadMultiplyer(Matrix firstMatrix, Matrix second
             filler.fillMatrixFromVector(thirdMatrix, futureVector.get(), x);
         }
         executor.shutdown();
-        thirdMatrix.showMatrix();
+
         return thirdMatrix;
     }
 
     private static void writeFile(FileHandler handler, FileParser parser, String path, Matrix resultMatrix) throws IOException {
-
         File file = handler.createFile(path);
```

Comment:
check, can you write file if remove variable "file" and use like this handler.writeFile(file, parser.parseVariablesToList(resultMatrix));